modules/mobile/docs/index.html

```diff
@@ -1078,7 +1078,7 @@
,"security":[{"Bearer":[]}],"parameters":[{"in":"path","name":"endpointSid","required":true,"schema":{"type":"string"},"description":"device endpointSid provided by the register endpoint"}],"responses":{"200":{"description":"OK","content":{"application/json":{"schema":{"type":"object","required":["data"],"properties":{"data":{"type":"object","required":["type","id","attributes"],"properties":{"type":{"type":"string","example":"pushGetPrefs"},"id":{"type":"string","example":"A3646A8D40C5B7319416179833809496","description":"Endpoint SID from request"},"attributes":{"type":"object","required":["preferences"],"properties":{"preferences":{"type":"array","items":{"type":"object","required":["preferenceId","preferenceName","value"],"properties":{"preferenceId":{"type":"string","example":"appointment_reminders"},"preferenceName":{"type":"string","example":"Appointment Reminders"},"value":{"type":"boolean","example":true}}}}}}}}}}}}},"401":{"description":"Authentication is possible but has failed or not yet been provided.","content":{"application/json":{"schema":{"$ref":"#/paths/~1/post/responses/408/content/application~1json/schema"}}}},"404":{"description":"The requested resource could not be found.","content":{"application/json":{"schema":{"$ref":"#/paths/~1/post/responses/408/content/application~1json/schema"}}}},"408":{"description":"The response from this API timed out.","content":{"application/json":{"schema":{"$ref":"#/paths/~1/post/responses/408/content/application~1json/schema"}}}},"422":{"description":"Failure in processing entity","content":{"application/json":{"schema":{"$ref":"#/paths/~1/post/responses/408/content/application~1json/schema"}}}},"500":{"description":"An internal API error occured.","content":{"application/json":{"schema":{"$ref":"#/paths/~1/post/responses/408/content/application~1json/schema"}}}},"502":{"description":"An upstream service the API depends on returned an error.","content":{"application/json":{"schema":{"$ref":"#/paths/~1/post/responses/408/content/application~1json/schema"}}}},"503":{"description":"An upstream service is unavailable, or its circuit breaker may have been tripped.","content":{"application/json":{"schema":{"$ref":"#/paths/~1/post/responses/408/content/application~1json/schema"}}}},"504":{"description":"A response from an upstream service timed out.","content":{"application/json":{"schema":{"$ref":"#/paths/~1/post/responses/408/content/application~1json/schema"}}}}}},"put":{"summary":"/v0/push/prefs/{endpointSid}","description":"Set the user's push notification preferences","operationId":"putPrefs","security":[{"Bearer":[]}],"parameters":[{"in":"path","name":"endpointSid","required":true,"schema":{"type":"string"},"description":"device endpointSid provided by the register endpoint"}],"requestBody":{"description":"Push notification preferences","required":true,"content":{"application/json":{"schema":{"type":"object","required":["preference","enabled"],"properties":{"preference":{"type":"string","examp
```

Comment:
I might have to take your word on this one. :)

Reply:
Yeah, the html is massive. The real changes happen in the docs yml files. In this case it was User.yml